app/src/main/res/values/strings.xml

```diff
@@ -946,6 +946,18 @@
     <string name="gen_sq_text">To generate Square wave</string>
     <string name="gen_pwm_text">Similarly, to produce four different PWM signal</string>
 
+    <string name="faq_question1">How can I use this application?</string>
+    <string name="faq_question2">What all experiments can I perform?</string>
+    <string name="faq_question3">Where can I find the schematic for connections of the devices?</string>
+    <string name="faq_question4">How can I design my own experiment and save it?</string>
+    <string name="faq_question5">Can I submit a change request for a particular instrument? If so, how?</string>
+
+    <string name="faq_answer1">Each device includes a bottomsheet guide which gets open when you slide up from the bottom of your screen. It includes all the necessary details to connect the device along a brief guide to use it.</string>
+    <string name="faq_answer2">The goal of PSLab is to create an Open Source hardware device (open on all layers) that can be used for experiments by teachers, students and citizen scientists. Our tiny pocket lab provides an array of sensors for doing science and engineering experiments. It provides functions of numerous measurement devices including an oscilloscope, a waveform generator, a frequency counter, a programmable voltage, current source and as a data logger. Hence, you can perform all kind of experiments using these devices.</string>
+    <string name="faq_answer3">The bottomsheet guide implemented in each device has all the schematics for the connection of the device.</string>
+    <string name="faq_answer4">Right now, we are not having an option to make and save custom experiments. But we will implement it soon.</string>
+    <string name="faq_answer5">You can submit your reviews, feature requests and feedback through our Feedback and Issues page.</string>
```

Comment:
Just a thought, may be we can group question and answer together.

Reply:
Actually I tried doing it but in expandable view it wasn't getting supported. Hence I went with this.